slash/plugins/builtin/xunit.py

```diff
@@ -48,10 +48,18 @@ def error_added(self, result, error):
 
     def _add_error(self, errortype):
         exc_type, exc_value, exc_tb = exc_info = sys.exc_info()
+        self._add_test_metadata(errortype,
+                                dict(type=exc_type.__name__,
```

Comment:
General style note: `{'type': exc_type.__name__, 'message': str(exc_value)}` is more widely accepted for dictionary initialization and is also slightly faster